README.md

```diff
@@ -22,24 +22,24 @@ To quickly get started, follow the installation instructions for [Python][1] or
+| `addExtension`         | Whether to install the Datadog Lambda Extension as a layer. Defaults to `false`. When enabled, it's required to set the `apiKey` (or `apiKMSKey`) parameter. While using `addExtension` leave out `forwarderArn`. The Datadog Lambda Extension Layer is in public preview. Learn more about the Lambda Extension Layer [here][8].                                                                                   |
+| `logLevel`             | The log level, set to `DEBUG` for extended logging. Defaults to `info`.                                                                                                                                                                                                                                                                                                                                             |
+| `enableXrayTracing`    | Set `true` to enable X-Ray tracing on the Lambda functions and API Gateway integrations. Defaults to `false`.                                                                                                                                                                                                                                                                                                       |
+| `enableDDTracing`      | Enable Datadog tracing on the Lambda function. Defaults to `true`.                                                                                                                                                                                                                                                                                                                                                  |
+| `enableApiGatewayLogs` | Enable automatic subscription of the Datadog Forward to API Gateway log groups. Defaults to `true`.                                                                                                                                                                                                                                                                                                                 |
+| `forwarderArn`         | Setting this parameter subscribes the given Datadog forwarder to the Lambda functions’ CloudWatch log groups. Required when `enableDDTracing` is set to `true` unless the subscription is otherwise applied. For example, if a Datadog Forwarder subscription is applied via Datadog's AWS Integration, then `forwarderArn` is not required. Do not set `addExtension` to `true` when using `forwarderArn`.         |
```

Comment:
Moving forward, we are going to want customers using the Lambda Extension to send Lambda Logs to Datadog. But if they want API Gateway logs they would need the Forwarder as well for that. So, we would want to enable subscriptions for the Forwarder to the API Gateway logs but *not* use the Forwarder for the Lambda logs. I think there are probably some code/documentation updates needed to cover this situation.

Reply:
For example, right now you should not set both `forwarderArn` and `addExtension`. But now you might actually want to do that.

Reply:
@DarcyRaynerDD is this something I should implement now?